OpenRA.Mods.Common/Traits/Render/WithInfantryBody.cs

```diff
@@ -159,31 +161,27 @@ protected virtual void Tick(Actor self)
 			dirty = false;
 		}
 
+		void INotifyBecomingIdle.OnBecomingIdle(Actor self)
+		{
+			if (state != AnimationState.Waiting)
```

Comment:
Can you remember your motivation here? This cuts off attacking animations before they can complete, and I don't notice anything immediately wrong if I comment it out.

Reply:
Not really. I'll remove it for now, if we notice a regression later we'll at least know where to look.